concoction-lib/src/test/java/info/mmpa/concoction/scan/InstructionMatchingTests.java

```diff
@@ -25,7 +26,10 @@ public class InstructionMatchingTests {
 	@Test
 	void test() {
 		try {
-			Results results = results("Runtime_exec.json", RuntimeExec.class);
+			Results results = Concoction.builder()
+					.model(Paths.get("src/test/resources/models/Runtime_exec.json"))
+					.primarySource(TestUtils.appModel(RuntimeExec.class).primarySource())
+					.scan();
```

Comment:
This is not what I intended by the prior comment talking about separation of duties

Reply:
what did you intend? I thought you meant this